index.css

```diff
@@ -0,0 +1,161 @@
+html,
+body {
+    margin: 0;
+    font-family: 'Karla', sans-serif;
+    background-color: #219ebc;
+}
+
+/*typography*/
+
+h1,
+h2,
+h3 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #EED6D3;
+    margin: 0;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+h2 {
+    font-size: 16px;
+}
+
+h3 {
+    margin: 25px;
+}
+
+/* layout */
+
+header {
+    background-color: #67595E;
+}
+
+.header-inner {
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    width: 370px;
+    gap: 15px;
+    padding: 10px 0 0 0;
+}
+
+.pumpkin-img {
+    width: 80px;
+}
+
+.controls-container {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    margin: 0px auto;
+}
+
+/* controls */
+
+.emotion-radios {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #bcbcbc;
+    border-radius: 5px;
+}
+
+.radio {
+    display: flex;
+    align-items: center;
+    border-bottom: 1px solid #bcbcbc;
+    padding: 4px 8px;
+}
+
+label {
+    font-size: 14px;
+    padding: 3px;
+    width: 100%;
+    cursor: pointer;
+}
+
+.highlight {
+    background-color: #fff0ee;
+    color: #bd301d;
+    accent-color: #bd301d;
+    font-weight: bold;
+}
+
+.radio:last-child {
+    border-bottom: 0;
+}
+
+input[type="radio"] {
+    cursor: pointer;
+}
+
+.gifs-check {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 16px;
+}
+
+.gifs-check-label {
+    width: unset;
+    margin-right: 5px;
+}
+
+.get-image-btn {
+    background-color: #67595e;
+    color: #eed6d3;
+    padding: 15px 19px;
+    font-size: 24px;
+    border: none;
+    border-radius: 5px;
+    margin-bottom: 8px;
+}
+
+/* modal */
+
+.meme-modal {
+    display: none;
+    /*flex when displayed*/
+    position: fixed;
+    margin: auto;
+    top: 80px;
+    left: 0;
+    right: 0;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #67595e;
+    border-radius: 5px;
+    width: 420px;
```

Comment:
instead of an explicit width/height, I'd allow the content to set the width/height:

```css
width: fit-content;
padding: 50px;
```

Reply:
I initially added a padding to the .cat-img class which accomplished the same thing but for some reason the border-radius was no longer applied to the image. Why is this? I updated to include a padding in the .meme-modal class and removed the padding in the .cat-img class and that brought back the border-radius and centered it in the container.